### PR TITLE
Update for 23 1

### PR DESCRIPTION
**Pull request by khamlyuk**

### CODE REVIEW

File: Readme.md [hunk 1]
```diff
@@ -3,12 +3,24 @@
...
-<!-- default file list end -->
+# Blazor Form Layout - Change visibility of the DxFormLayout's items and groups conditionally
 
-### Blazor Form Layout - How to change the visibility of DxFormLayout's items and groups
+The following example shows how to implement a form that dynamically displays additional inputs based on user choice.
 
-This demo illustrates how to implement a form that dynamically exposes different additional inputs depending upon the initial selection made by an end-user. The visibility of these additional inputs is specified using the Visible property of the DxFormLayoutGroup and DxFormLayoutItem elements. 
+![Final Layout](/result.png)
+
+A user chooses one of the options in Radio Group. Based on selected value, the [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.FormLayoutItemBase.Visible) property of the `DxFormLayoutGroup` and `DxFormLayoutItem` UI elements changes and displays or hides additional inputs.
```
Comment (albertov05) [inline Readme.md]:
```suggestion
A user chooses one of the options in a radio group. Based on the selected value, the [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.FormLayoutItemBase.Visible) property of the `DxFormLayoutGroup` and `DxFormLayoutItem` UI elements changes and displays or hides additional inputs.
```

File: Readme.md [hunk 2]
```diff
@@ -3,12 +3,24 @@
...
+
+[Index.razor](./CS/DxFormLayoutSelectionDependentInputs/Pages/Index.razor)
+
+## Documentation
+
+* [DxRadioGroup](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxRadioGroup-2.ValueChanged)
+* [DxFormLayout](https://docs.devexpress.com/Blazor/DevExpress.Blazor.DxFormLayout)
+
+## More Examples
+
```
Comment (albertov05) [inline Readme.md:25]: In the result.png image below, it shows both "E-mail" and "Email". I think either spelling is fine, but can you please make it consistent?

File: Readme.md (hunk 1 above)
Comment (khamlyuk) [reply Readme.md]: "Radio Group" is a component name here.

```suggestion
A user chooses one of the options in a Radio Group. Based on the selected value, the [Visible](https://docs.devexpress.com/Blazor/DevExpress.Blazor.Base.FormLayoutItemBase.Visible) property of the `DxFormLayoutGroup` and `DxFormLayoutItem` UI elements changes and displays or hides additional inputs. 
```

File: Readme.md (hunk 2 above)
Comment (khamlyuk) [reply Readme.md:25]: Good point, thank you!

**Pull request merged**